### PR TITLE
Toast component

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my toast components

this page contains the toast components of Component library

live link: https://componentsofcss.netlify.app/component%20html/toast.html

Following are the features of this page

nav: contains brand name and links to the home page and components library.
main: contains the documentation of toast components with snippets and examples.
aside: contains links to all my component libraries.
footer: contains links to my social media

image

image

### CODE REVIEW

**Review by parulgupta2001 — COMMENTED** [review]
add color as variable

**Review by parulgupta2001 — APPROVED** [review]

File: utkarsh component/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+
+.container{
+    color: var(--primary-color);
+    width: 100vw;
+    text-decoration: none;
+    display: grid;
+    font-weight: 400;
+    grid-auto-flow: column;
+    grid-template-columns: 200px 1fr; 
+
```
Comment (khushijohri2001) [inline utkarsh component/toast/toast.css]: you can remove the extra void line here

File: utkarsh component/toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+color: var(--secondary-color);
+}
+
+.current-pg{
+    color: var(--secondary-color);
+    display: block;
+        background-color: #eaf1dd;
+        /* color: whitesmoke; */
```
Comment (khushijohri2001) [inline utkarsh component/toast/toast.css]: you can remove this comment if no needed

File: utkarsh component/toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,625 @@
...
+}
+
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
+    
+    /* grid-template-areas: 200px 1fr; */
```
Comment (khushijohri2001) [inline utkarsh component/toast/toast.css]: here too

**Review by khushijohri2001 — APPROVED** [review]

File: utkarsh component/toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,625 @@
...
+        top: 0.3rem;
+      }
+      
+      .bars::after {
+        top: -0.3rem;
+      }
+      
+      /* Tablet Menu */
+      @media all and (min-width: 468px) {
+        .menu {
```
Comment (i-am-a-codegirl) [inline utkarsh component/toast/toast.css]: kindly do not  use media queries anywhere in the middle of your CSS and always ensure that all of them are arranged in the  end of your CSS file

**Review by i-am-a-codegirl — APPROVED** [review]

**Pull request merged**